scene/animation/tween.cpp

```diff
@@ -594,7 +594,7 @@ PropertyTweener::PropertyTweener(Object *p_target, NodePath p_property, Variant
 }
 
 PropertyTweener::PropertyTweener() {
-	ERR_FAIL_MSG("Can't create empty PropertyTweener. Use get_tree().tween_property() or tween_property() instead.");
+	ERR_FAIL_MSG("PropertyTweener can't be created directly. Use get_tree().create_tween().tween_property() instead.");
```

Comment:
```suggestion
	ERR_FAIL_MSG("PropertyTweener can't be created directly. Use get_tree().create_tween().tween_property() or create_tween().tween_property() instead.");
```
It should be more like this, if following the previous message.

Reply:
Too verbose IMO, these long error messages are hard to read - How about "PropertyTweener can't be created directly. Use the tween_property() method in Tween."?

Reply:
Sounds good.